tests/upgrade_path/test_upgrade_path.py

```diff
@@ -10,10 +10,13 @@
 from datetime import datetime
 
 from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory   # lgtm[py/unused-import]
-from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import remove_ip_addresses      # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py     # lgtm[py/unused-import]
 
 pytestmark = [
-    pytest.mark.topology('any')
+    pytest.mark.topology('any'),
+    pytest.mark.sanity_check(skip_sanity=True),
```

Comment:
Why are we skipping sanity check?

Reply:
Because the first step in test_upgrade_path is to install a base image, and then perform a cold-reboot, which probably recover the DUT from all kinds of errors. That's why I disable sanity_check here.